.github/autobuild/linux_deb.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -eu
+
+if [[ ! ${jamulus_buildversionstring:-} =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    echo "Environment variable jamulus_buildversionstring has to be set to a valid version string"
+    exit 1
+fi
+
+setup() {
+    echo "Update system..."
+    sudo apt-get -qq update
+
+    echo "Install dependencies..."
+    sudo apt-get -qq --no-install-recommends -y install devscripts build-essential debhelper libjack-jackd2-dev qtbase5-dev qttools5-dev-tools
+}
+
+build() {
+    ./linux/deploy_deb.sh
```

Comment:
Hm, didn't think of that. I guess it would work, however, it might cause surprises later. If we added something after the switch statement (some hypothetical `echo "Finished successfully"`), it wouldn't run. I think I'd rather keep it as-is?

Reply:
OK, I wasn't sure about the intent.